libs/ui/src/with_scroll_buttons.tsx

```diff
@@ -0,0 +1,203 @@
+/* stylelint-disable order/properties-order */
+/* istanbul ignore file - tested via VxMark cypress tests */
+import React from 'react';
+import styled, { ThemeProvider } from 'styled-components';
+
+import { rgba } from 'polished';
+import { SizeMode } from '@votingworks/types';
+import { Button } from './button';
+import { Icons } from './icons';
+import { makeTheme } from './themes/make_theme';
+
+export interface WithScrollButtonsProps {
+  children: React.ReactNode;
+}
+
+const SCROLL_DISTANCE_TO_CONTENT_HEIGHT_RATIO = 0.75;
+
+const SCROLL_BUTTON_SIZE_MODE_OVERRIDES: Readonly<Record<SizeMode, SizeMode>> =
+  {
+    s: 's',
+    m: 'm',
+    l: 'm',
+    xl: 'm',
+    legacy: 'legacy',
+  };
```

Comment:
What's the VVSG rationale here? I agree it seems totally necessary, but want to understand if we have an explicit VVSG carveout or not.

Reply:
Yeah, nothing specifically around navigation/scroll controls, so this is more to keep the button sizes tame in our specific UI. WRT text size, VVSG does allow for some text elements to be smaller than the currently active text size, as long as crucial elements like candidate names are rendered at the selected size.

I felt we could've gotten by with capping it at `L` instead of `M`, but it falls apart in the ms-either-neither case (not sure if that contest is something we'll support long-term in its current format)